Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -2024,18 +2024,17 @@ - (void)didTapAudioViewItem:(ConversationViewItem *)viewItem attachmentStream:(T
     [self.audioAttachmentPlayer play];
 }
 
-- (void)didTapOversizeTextMessage:(NSString *)displayableText attachmentStream:(TSAttachmentStream *)attachmentStream
+- (void)didTapTruncatedTextMessage:(ConversationViewItem *)conversationItem
 {
     OWSAssert([NSThread isMainThread]);
-    OWSAssert(displayableText);
-    OWSAssert(attachmentStream);
+    OWSAssert(conversationItem);
 
-    // Tapping on incoming and outgoing "oversize text messages" should show the
-    // "oversize text message" view.
-    OversizeTextMessageViewController *messageVC =
```

Comment:
Got rid of `OversizeTextMessageViewController` in favor of `MessageMetadataViewController`